installer/PowerToysSetup/generateFileList.ps1

```diff
@@ -0,0 +1,84 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory = $True, Position = 1)]
+    [AllowEmptyString()]
+    [string]$fileDepsJson,
+    [Parameter(Mandatory = $True, Position = 2)]
+    [string]$fileListName,
+    [Parameter(Mandatory = $True, Position = 3)]
+    [string]$wxsFilePath,
+    # If there is no deps.json file, just pass path to files
+    [Parameter(Mandatory = $False, Position = 4)]
+    [string]$depsPath,
+    # launcher plugins are being loaded into launcher process,
+    # so there are some additional dependencies to skip
+    [Parameter(Mandatory = $False, Position = 5)]
+    [bool]$isLauncherPlugin,
+    # Skip winAppSDK dlls as those are hard-linked
+    [Parameter(Mandatory = $False, Position = 6)]
+    [bool]$isWinAppSdkProj
+)
+
+$fileWxs = Get-Content $wxsFilePath;
+
+#Skip PowerToysInterop files
+$coreWxs = Get-Content $PSScriptRoot/"Core.wxs"
+$coreWxs | ForEach-Object {
+    if ($_ -match "(<?define PowerToysInteropFiles=)(.*)\?>") {
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', 'fileList',
+        Justification = 'variable is used in another scope')]
+
+        $interopFilesList = $matches[2] -split ';'
+        return
+    }
+}
+
+#Skip WinAppSdk files
+if ($isWinAppSdkProj -eq $True) {
+    $winAppSDKWxs = Get-Content $PSScriptRoot/"WinAppSDK.wxs"
+    $winAppSDKWxs | ForEach-Object {
+        if ($_ -match "(<?define WinAppSDKFiles=)(.*)\?>") {
+            [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', 'fileList',
+            Justification = 'variable is used in another scope')]
+
+            $winAppSDKfilesList = $matches[2] -split ';'
+            return
+        }
+    }
+}
+
+$fileExclusionList = @("*Test*", "*.pdb", "*.lastcodeanalysissucceeded", "createdump.exe", "backup_restore_settings.json") + $interopFilesList + $winAppSDKfilesList
```

Comment:
```suggestion
$fileExclusionList = @("*Test*", "*.pdb", "*.lastcodeanalysissucceeded", "createdump.exe", "backup_restore_settings.json", $interopFilesList, $winAppSDKfilesList);
```

Reply:
this doesn't work for some reason. Reverted 🤷

Reply:
@stefansjfw 🤔🤔🤔 hmm that's weird.  Oh well, the previous way also works :) .